core/src/main/scala/akka/kafka/RestrictedConsumer.scala

```diff
@@ -36,6 +37,7 @@ final class RestrictedConsumer(consumer: Consumer[_, _], duration: java.time.Dur
   /**
    * See [[org.apache.kafka.clients.consumer.KafkaConsumer#committed(TopicPartition,java.time.Duration)]]
    */
+  @silent
   def committed(tp: TopicPartition): OffsetAndMetadata = consumer.committed(tp, duration)
```

Comment:
Perhaps this should be deprecated as well.

Reply:
If `consumer.committed` is deprecated, we should deprecate as well.

Reply:
I'll make a follow up PR that adds the replacement and deprecates this instance.

Reply:
https://github.com/akka/alpakka-kafka/pull/1178